### ISSUE TITLE
Investigate electronic signing of CLAs

### ISSUE THREAD

**Issue by iteles** [dwyl/summer-2015#29]
https://cla.developers.google.com/about/google-individual

Linked to #27

**Comment by iteles** [comment]
A few options:
- There are of course **many paid eSignature services**, for example:
  - [HelloSign](https://www.hellosign.com/info/pricing)
  - [DocuSign](https://www.docusign.com/products-and-pricing)
  - [Adobe's EchoSign](https://acrobat.adobe.com/us/en/pricing/business-pricing.html)
- There are also some **free ones** but I'm not expecting people to be keen on handing over their signatures to just any app:
  - [CudaSign](https://itunes.apple.com/us/app/signnow-sign-fill-pdf-word/id489262811) for iOS and [DocuSign](https://play.google.com/store/apps/details?id=com.docusign.ink&hl=en) for Android
  - [eSign Genie](https://www.esigngenie.com/)
- Google has a simple **'I agree' button** on [their form](https://cla.developers.google.com/about/google-individual) without requiring an actual digital signature
  <img width="700" alt="google-cla-signing" src="https://cloud.githubusercontent.com/assets/4185328/8545971/82c03a4e-24ab-11e5-8920-20909ec245cd.png">
- **PDF signatures** using [Preview for Mac](http://computers.tutsplus.com/tutorials/quick-tip-digitally-insert-signatures-into-documents-using-preview--mac-44429) or [Adobe Reader for Windows](https://helpx.adobe.com/reader/using/sign-pdfs.html)
- **Good ol' fashioned paper** & pen

Some more investigation is required on the legal implications of the **Google 'I agree' button**, but this seems like it might be the simplest and **most intuitive solution long term**.    

**tl;dr**
We ultimately [don't want to be giving out people's data](https://github.com/dwyl/start-here/blob/master/manifesto.md) to 'third parties' we (or developers who might want to collaborate with us) don't trust.    

For the **short term** (summer), we should choose between paper agreements or PDF signatures (which I would favour as they're more eco-friendly).

Any objections to this? (cc/ @nelsonic)

**Comment by nelsonic** [comment]
Agree with not wanting to give out people's personal data. I don't want to _store_ it in the first place.
I quite like the google "I agree" approach, they know they won't have to legally enforce it.
I've used _all_ of the paid ones while @GRPN so know the pros/cons of each. Hello sign is the simplest.
Ultimately I want to _formalised_ gentleman's agreement, more of a social contract than a market one,
more Kiva or Grameen than Barclays ... 

Does anyone else have thoughts on this? how would @dwyl/doers do it?

**Comment by iteles** [comment]
Right, but for this summer, even though I don't want _us_ to store them, I think we have much better **data privacy principles** than third parties...

**Comment by nelsonic** [comment]
Agree. so... _conclusion_?

**Comment by iteles** [comment]
PDFs for the summer, 'I agree' button after that was my tl;dr!

**Comment by nelsonic** [comment]
PDFs which get _physically_ printed off and signed? or...?

**Comment by iteles** [comment]
PDFs which are signed digitally using Preview or Adobe Reader annotating functions. :green_heart: :evergreen_tree:
